db/migrate/20160825122452_remove_duplicate_drafts.rb

```diff
@@ -0,0 +1,34 @@
+class RemoveDuplicateDrafts < ActiveRecord::Migration
+  def up
+    content_items = ContentItem.where(id: [815434, 818621])
+
+    if content_items.any? { |x| x.updated_at != x.created_at }
+      raise "content item #{x.id} has been changed"
```

Comment:
As this is deleting the content items, if you rerun the migration, it does nothing as no content items are found with those ids.

These errors will only happen if the migration should not be run when run in production because the assumptions under which I wrote it do not match the database.

I can still change it just to puts and return, but having it fail and abort the transaction seems better.